components/PrimaryCard.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import Svg, { Circle, Path, LinearGradient, Stop, Defs } from 'react-native-svg';
+import { PathProgressCardStyles } from '@styles';
+import { Constants } from '@constants';
+
+interface Prop {
+  sehajPathName: string;
+  angNumber: number;
+  progress: number;
+  onPress: () => void;
+}
+
+export const PrimaryCard = ({ sehajPathName, angNumber, progress, onPress }: Prop) => {
+  progress = progress >= 100 ? 99 : progress;
```

Comment:
This feels weird to read. Why are we setting the progress specifically to 99 here. 

Similar for code following it. Consider using some constant variables here, with descriptive names instead of hardcoded values directly in code.